builtin/settingtypes.txt

```diff
@@ -468,9 +468,39 @@ smooth_lighting (Smooth lighting) bool true
 #    at the expense of minor visual glitches that do not impact game playability.
 performance_tradeoffs (Tradeoffs for performance) bool false
 
+[**Advanced]
+
 #    Adds particles when digging a node.
 enable_particles (Digging particles) bool true
 
+#    Decide the color depth of the texture used for the post-processing pipeline.
+#    Reducing this can improve performance, but might cause some effects (e.g. bloom)
+#    to not work.
+#
+#    Requires: enable_post_processing
+post_processing_texture_bits (Color depth for post-processing texture) enum 16 8,10,16
+
+#    Enable Poisson disk filtering.
+#    On true uses Poisson disk to make "soft shadows". Otherwise uses PCF filtering.
+#
+#    Requires: enable_dynamic_shadows, opengl
+shadow_poisson_filter (Poisson filtering) bool true
+
+#    Spread a complete update of shadow map over given number of frames.
+#    Higher values might make shadows laggy, lower values
+#    will consume more resources.
+#    Minimum value: 1; maximum value: 16
+#
+#    Requires: enable_dynamic_shadows, opengl
+shadow_update_frames (Map shadows update frames) int 8 1 16
+
+#    Set the soft shadow radius size.
+#    Lower values mean sharper shadows, bigger values mean softer shadows.
+#    Minimum value: 1.0; maximum value: 15.0
+#
+#    Requires: enable_dynamic_shadows, opengl
+shadow_soft_radius (Soft shadow radius) float 5.0 1.0 15.0
```

Comment:
advanced settings need to be put into a section in the "Advanced" category so they're actually less visible / hidden when advanced settings are hidden

Reply:
settingtypes.txt says:
```
# There shouldn't be too much settings per category; settings that shouldn't be
#  modified by the "average user" should be in (sub-)categories called "Advanced".
```

if this is incorrect then we need to fix the documentation.

Reply:
Yeah, that comment is wrong. The current advanced settings structure was introduced in #12490 and implemented in #13861. The "Client and Server", "Mapgen" and "Advanced" top-level categories are the ones containing advanced settings.

Reply:
then this category doesn't work either I guess
https://github.com/minetest/minetest/blob/df4e70b2c7631317430bd78061507270d23db67b/builtin/settingtypes.txt#L1821-L1829

Reply:
That's the "Advanced" category in the "Advanced" top-level category, so it works.

(By "top-level category", I mean the headings without asterisks (-> thus having the highest level in the category hierarchy). These are shown as yellow labels in the GUI.)

Reply:
I discovered further unclear behavior of the advanced toggle but the settings are now moved.